server/views/applications/pages/health-needs/guidance.njk

```diff
@@ -0,0 +1,36 @@
+{% extends "../layout.njk" %}
+{% block questions %}
+  <h1 class="govuk-heading-l govuk-!-margin-bottom-4">{{ page.title }}</h1>
+
+  <p class="govuk-body">
+    To complete this section, you’ll need to enter health information about the applicant.
+  </p>
+
+  <p class="govuk-body">
+    Typically, this <strong>could</strong> involve speaking to the following people:
+  </p>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <li>Healthcare team</li>
+    <li>Substance/Drugs team</li>
+    <li>Substance mis-use agency</li>
+    <li>Mental Health team</li>
+    <li>Adult social care</li>
+    <li>Nelson Trust</li>
+  </ul>
+
+  <h2 class="govuk-heading-m">Getting in touch</h2>
+
+  <p class="govuk-body">
+    <strong>We strongly suggest you get in touch with the relevant health teams and officers now.</strong>
+  </p>
+
+  <p class="govuk-body">
```

Comment:
minor question but I'm wondering if we should not include this paragraph and the below one? The user doesn't 'need' them and on the off chance we don't get around to doing the story to insert the copy/paste text then we would have to come back and remove these paragraphs...

Reply:
yes, makes sense  -- certainly it's neater.